app/models/family.rb

```diff
@@ -1,19 +1,21 @@
 class Family < ApplicationRecord
+  has_paper_trail
+
   include OrganizationScope
 
-  belongs_to :male, class_name: 'Animal'
-  belongs_to :female, class_name: 'Animal'
+  belongs_to :male, class_name: 'Animal', optional: true
+  belongs_to :female, class_name: 'Animal', optional: true
 
   has_many :measurements
 
   belongs_to :tank, required: false
 
   def female_pii_tag
-    female.pii_tag
+    female&.pii_tag
```

Comment:
Due to above comment.

Reply:
have you considered a delegate here @haydenrou ?

```suggestion
    delegate :pii_tag, to: :female, prefix: true, allow_nil: true
```

Reply:
Good idea, I was initially just fixing the broken page but i'll make this refactor too